### PR TITLE
@dblandin => Adjusted zero state

### PR DESCRIPTION
**Pull request by mzikherman**
W/o any tabs or totals when there are no results:

Screen Shot 2019-04-04 at 1 50 54 PM

We also are hiding tabs if there are no results of that type.

Closes https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=57&projectKey=DISCO&modal=detail&selectedIssue=DISCO-881

### CODE REVIEW

File: src/Apps/Search/Components/NavigationTabs.tsx [hunk 1]
```diff
@@ -80,40 +91,18 @@ export class NavigationTabs extends React.Component<Props> {
...
-    const articlesAggregationCount = get(
-      this.aggregationFor("article"),
-      agg => agg.count,
-      0
-    )
-    const auctionsAggregationCount = get(
-      this.aggregationFor("sale"),
-      agg => agg.count,
-      0
+    const tabCountMap = Object.entries(TAB_NAME_MAP).reduce(
```
Comment (mzikherman) [inline src/Apps/Search/Components/NavigationTabs.tsx]: This is all a refactor to DRY up the tab/count rendering, and also this now doesn't render tabs where the counts are 0l

File: src/Apps/Search/SearchApp.tsx [hunk 2]
```diff
@@ -24,6 +25,48 @@ export interface Props {
   context_page: Schema.PageName.SearchPage,
 })
 export class SearchApp extends React.Component<Props> {
+  renderResults(count: number, artworkCount: number) {
```
Comment (mzikherman) [inline src/Apps/Search/SearchApp.tsx]: Adjusted the app to display the zero state (w/o rendering the '0 results for andy' header, or any tabs).

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit d0faf68f7b2c7cab8c0574aac914ef7002ff353e

https://deploy-preview-2257--artsy-reaction.netlify.com

**Comment by mzikherman** [comment]
Merging!

**Comment by artsyit** [comment]
:rocket: PR was released in v15.11.24 :rocket:

**Pull request merged**